package.json

```diff
@@ -35,6 +35,10 @@
       "require": "./dist/esbuild.js",
       "import": "./dist/esbuild.mjs"
     },
+    "./ts-diagnostic": {
+      "default": "./dist/ts-diagnostic.js",
```

Comment:
`ts-diagnostic` uses `import`, so this doesn't work in CJS repos (e.g. I tried in one of the unplugin examples). We need to build an `.mjs` version and link to here.